index 611c3c0..5951970 100644
--- a/CCRakNetSlidingWindow.cpp
+++ b/CCRakNetSlidingWindow.cpp
@@ -22,7 +22,7 @@ static const CCTimeType SYN=10000;
 
 #include "MTUSize.h"
 #include <stdio.h>
-#include <math.h>
+#include <cmath>
 #include <stdlib.h>
 #include "RakAssert.h"
 #include "RakAlloca.h"
@@ -218,7 +218,7 @@ void CCRakNetSlidingWindow::OnAck(CCTimeType curTime, CCTimeType rtt, bool hasBA
 		double d = .05;
 		double difference = rtt - estimatedRTT;
 		estimatedRTT = estimatedRTT + d * difference;
-		deviationRtt = deviationRtt + d * (abs(difference) - deviationRtt);
+		deviationRtt = deviationRtt + d * (std::abs(difference) - deviationRtt);
 	}
 
 	_isContinuousSend=isContinuousSend;
