index c116ee5..567c044 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,7 +17,7 @@ IF(WIN32 AND NOT UNIX)
 
 	IF(NOT ${CMAKE_GENERATOR} STREQUAL "MSYS Makefiles")
 
-		IF( MSVC10 OR MSVC11 OR MSVC12 )
+		IF( MSVC10 OR MSVC11 OR MSVC12 OR MSVC14 OR MSVC15 )
 			set_target_properties(RakNetLibStatic PROPERTIES STATIC_LIBRARY_FLAGS "/NODEFAULTLIB:\"LIBCD.lib LIBCMTD.lib MSVCRT.lib\"" )
 		ELSE()
 			set_target_properties(RakNetLibStatic PROPERTIES STATIC_LIBRARY_FLAGS "/NODEFAULTLIB:&quot;LIBCD.lib LIBCMTD.lib MSVCRT.lib&quot;" )
